package.json

```diff
@@ -37,5 +40,12 @@
   },
   "peerDependencies": {
     "rxjs": "^6.3.3"
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/commit-analyzer",
```

Comment:
I had to configure the commit analyzer when I did the implementation in ansi-to-react to support conventional commits. I think we'll have to do the same here.

https://github.com/nteract/ansi-to-react/blob/master/package.json#L50

Reply:
Whoops, good catch!